### DISCUSSION TITLE
Commitment to Maintenance

### DISCUSSION THREAD

**Discussion by port19x** [pystardust/ani-cli#1334] [General] [answered]
Since I've recently left the discord/matrix, I don't want anyone to confuse that departure for me no longer being interested in supporting ani-cli.

With this discussion post I'm committing to continue maintenance of ani-cli until at least end-of-year, but probably well into 2025.

I'd be interested in the plans of my fellow maintainers @justchokingaround @Derisis13 @71zenith @CoolnsX

Also, @ykhan21 is kind of the windows shadow maintainer. @ykhan21 if you read this, please tell us if you want to be a full maintainer, I can arrange that.

**Comment by 71zenith** [comment, marked as answer]
I'll be the last to leave :slightly_smiling_face:

**Comment by Derisis13** [comment]
For me, ani-cli development is probably my lowest priority responsibility, but since there's not much work to do, and I've gotten to like you all, I'm staying. If we need to work on the project in the next one and half year, I can probably chip in my usual part.
However if the project were take a new direction, I would most likely to be against it.

**Comment by ykhan21** [comment]
@port19x, sure, if you do not mind. Thanks.

**Reply by port19x** [reply]
Just saw this, I'll get in touch with @pystardust so he can invite you tomorrow.
I'll keep you updated

**Reply by port19x** [reply]
That should have done it, welcome aboard

**Comment by justchokingaround** [comment]
i just saw this, i'll keep maintaining ani-cli as long as it's alive and will fix bugs if any come up, sorry for the late response, the ping didn't go through
